app/views/courses/CoursesView.coffee

```diff
@@ -43,6 +46,16 @@ module.exports = class CoursesView extends RootView
     @courses = new CocoCollection([], { url: "/db/course", model: Course})
     @supermodel.loadCollection(@courses)
 
+    # TODO: Trim this section for only what's necessary
+    @hero = new ThangType
+    defaultHeroOriginal = ThangType.heroes.captain
+    heroOriginal = me.get('heroConfig')?.thangType or defaultHeroOriginal
+    @hero.url = "/db/thang.type/#{heroOriginal}/version"
+    # @hero.setProjection ['name','slug','soundTriggers','featureImages','gems','heroClass','description','components','extendedName','unlockLevelName','i18n']
+    @supermodel.loadModel(@hero, 'hero')
+    @listenTo @hero, 'all', ->
+      @render()
```

Comment:
I think this will trigger a ton of renders. Can the event listened to be just sync? If so, the supermodel is already listening for that and will render `onLoaded`. If it must be `'all'`, `render` should be debounced so that it only gets called once at the end of a stream of events.